okhttp/src/main/java/okhttp3/Request.java

```diff
@@ -241,6 +248,21 @@ public Builder method(String method, RequestBody body) {
       return this;
```

Comment:
I'd make this function just call the one you made: `method(method, body, false);`